hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java

```diff
@@ -1005,6 +1008,59 @@ public long getContainerCount() throws IOException {
     return response.getContainerCount();
   }
 
+  @Override
+  public List<HddsProtos.DatanodeDiskBalancerInfoProto> getDiskBalancerReport(
+      int count, int clientVersion) throws IOException {
+    DatanodeDiskBalancerInfoRequestProto request =
+        DatanodeDiskBalancerInfoRequestProto.newBuilder()
+            .setInfoType(DatanodeDiskBalancerInfoType.report)
+            .setCount(count)
+            .build();
+
+    DatanodeDiskBalancerInfoResponseProto response =
+        submitRequest(Type.DatanodeDiskBalancerInfo,
+            builder -> builder.setDatanodeDiskBalancerInfoRequest(request))
+            .getDatanodeDiskBalancerInfoResponse();
+
+    return response.getInfoList();
+  }
+
+  @Override
+  public List<HddsProtos.DatanodeDiskBalancerInfoProto> getDiskBalancerStatus(
```

Comment:
They return different contents, REPORT shows current volume density, while status show more data like "isRunning" and configuration.

Reply:
Both are returning DatanodeDiskBalancerInfoProto right now. Do you think we can have just one API for this and in the command line display different results for different commands?

Reply:
Internally, it's using one proto for STATUS and REPORT (DatanodeDiskBalancerInfoRequestProto).
On client side, since STATUS and REPORT needs different input params, merging them to one API might increase the complexity when builidng requests.

Reply:
From what I understand proto is still same for the request(DatanodeDiskBalancerInfoRequestProto). Status and report are just enums inside the request proto. Also I am not sure if we even require two different enum types: status and report here.

Reply:
I used two enum types because the responses are different for REPORT and STATUS.
For REPORT, it returns only current Volume Density Sum. For STATUS, it returns if it's running and the current configurations datanodes are using.

Reply:
I see. Makes sense. For Report you are not initialising configuration in DiskBalancerManager.